package.json

```diff
@@ -133,5 +133,8 @@
   "workspaces": [
     "packages/*",
     "app/"
-  ]
+  ],
+  "dependencies": {
```

Comment:
We should add this to the list of `devDependencies` like the other packages

Reply:
Missed that. I'll update that.